antora-playbook.yml

```diff
@@ -14,7 +14,7 @@ content:
     start_path: docs
     branches: [ develop, feature/SM16-31 ]
     tags:
-    - 1.*
+    - 1.0.0
```

Comment:
We should not use version `1.0.0` as there is no official release of the Handbook yet. It is to come in the future. To be confirmed (refer to our internal discussion).